core/src/components/datetime/test/show-adjacent-days/index.html

```diff
@@ -178,6 +178,14 @@ <h2>Change firstDayOfWeek</h2>
           <br /><br />
           <ion-datetime show-adjacent-days="true" id="display" value="2022-02-22T16:30:00"></ion-datetime>
         </div>
+
+        <ion-datetime
+          id="default2"
+          show-adjacent-days="true"
+          locale="en-US"
+          value="2022-10-14T16:22:00.000Z"
+          presentation="date"
+        ></ion-datetime>
```

Comment:
This needs to be inside a grid and have a title. I also recommend placing it after "Change firstDayOfWeek".